src/plugins/poster/poster.js

```diff
@@ -15,7 +15,8 @@ import $ from 'clappr-zepto'
 export default class PosterPlugin extends UIContainerPlugin {
   get name() { return 'poster' }
   get template() { return template(posterHTML) }
-  get shouldRender() { return this.container.playback.name !== 'html_img'}
+  get shouldRender() { return this.container.playback.name !== 'html_img' }
+  get shouldHideOnPlay() { return this.container.playback.name !== 'html_audio' }
```

Comment:
This isn't reliably working for me because I can't force HTML5Audio playback for a HLS audio-only stream. My last commit (manually specifying poster auto-hide behavior in options) worked fine for my use case, although this would be more elegant.